.github/workflows/main.yml

```diff
@@ -3,8 +3,12 @@ name: Hypothesis CI
 on:
   push:
     branches: [ master ]
+    paths:
+      - "!conjecture-rust/**"
   pull_request:
     branches: [ master ]
+    paths:
+      - "!conjecture-rust/**"
```

Comment:
I suspect that if both Rust and Python code change, this will skip Python (which we don't want).